_maps/map_files/stations/boxstation.dmm

```diff
@@ -64190,9 +64190,9 @@
 	icon_state = "4-8"
 	},
 /obj/machinery/camera{
-	c_tag = "Prison Solitary Confinement 2";
 	dir = 6;
-	network = list("Prison","SS13")
+	network = list("Prison","SS13");
```

Comment:
I have to ask - why is this camera, which is located down near engineering, a part of the prison camera network? It could probably make due with just being on the SS13 network.